### ISSUE TITLE
Cannot run on ADS 1.40.0 Universal build on MacOS M1

### ISSUE THREAD

**Issue by andershermansen** [microsoft/azuredatastudio-postgresql#335]
- Azure Data Studio Version: 1.40.0 (Universal)
- OS Version: macOS 12.6 (M1)

Steps to Reproduce:

1. Open Azure Data Studio
2. Install Postgresql extension

Actual:
Error `Failed to start Pgsql tools service`

Expected:
PostgreSQL extension loaded

Logs:
```
[2022-11-17 07:45:14.829] [exthost] [error] An error occurred when deactivating the subscriptions for extension 'Microsoft.azuredatastudio-postgresql':
[2022-11-17 07:45:14.831] [exthost] [error] TypeError: Cannot read properties of undefined (reading 'stop')
	at Object.dispose (/Users/anders/.azuredatastudio/extensions/microsoft.azuredatastudio-postgresql-0.2.7/out/main.js:101:68)
	at u (/Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:8:12323)
	at h._deactivate (/Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:88:13307)
	at /Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:88:10637
	at Array.map (<anonymous>)
	at h._deactivateAll (/Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:88:10625)
	at h.terminate (/Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:88:11009)
	at h.terminate (/Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:91:620)
	at d (/Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:101:36328)
	at Socket.<anonymous> (/Applications/Azure Data Studio.app/Contents/Resources/app/out/vs/workbench/api/node/extensionHostProcess.js:101:34024)
	at Socket.emit (node:events:402:35)
	at Pipe.<anonymous> (node:net:687:12)
```
[exthost.log](https://github.com/microsoft/azuredatastudio-postgresql/files/10028777/exthost.log)

**Comment by andershermansen** [comment]
Is it related to https://github.com/microsoft/azuredatastudio/issues/20775 ?

**Comment by akonyer** [comment]
I'm running into this as well just after updating Azure Data Studio to the ARM version on MacOS

**Comment by andershermansen** [comment]
This issue is fixed with [v0.3.0 release](https://github.com/microsoft/azuredatastudio-postgresql/releases/tag/v0.3.0).

@DaeunYim Thanks for the fixes and release.
